### ISSUE TITLE
problem ddl parse pattern

### ISSUE THREAD

**Issue by shinichi-takii** [shinichi-takii/ddlparse#13]
## Problem DDL Example
```sql
CREATE TABLE `example_table` (
  `id` bigint(20) NOT NULL AUTO_INCREMENT,
  `class_id` int(11) DEFAULT NULL,
  `group_id` int(11) DEFAULT NULL,
  `valid_date` date NOT NULL DEFAULT '0000-00-00',
  `count` int(11) DEFAULT NULL,
  `total` int(11) DEFAULT '0',
  `flag_1` int(11) DEFAULT '0',
  `balue_1` tinyint(1) DEFAULT '0',
  `flag_2` int(11) DEFAULT '0',
  `created_at` datetime DEFAULT NULL,
  `updated_at` datetime DEFAULT NULL,
  `lock_version` int(11) NOT NULL DEFAULT '0',
  PRIMARY KEY (`id`,`valid_date`),
  UNIQUE KEY `uniq_vd_cc_sg` (`valid_date`,`class_id`,`group_id`,`flag_2`),
  KEY `sg_cc_vd` (`group_id`,`class_id`,`valid_date`)
) ENGINE=InnoDB AUTO_INCREMENT=21619109476 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci
```

## Output BigQuery JSON Schema
```json
[{
	"name": "id",
	"type": "INTEGER",
	"mode": "REQUIRED"
}, {
	"name": "class_id",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "group_id",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "valid_date",
	"type": "DATE",
	"mode": "REQUIRED"
}]
```

**Comment by shinichi-takii** [comment]
fixed.

## Output BigQuery JSON Schema
### Normal
```json
[{
	"name": "id",
	"type": "INTEGER",
	"mode": "REQUIRED"
}, {
	"name": "class_id",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "group_id",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "valid_date",
	"type": "DATE",
	"mode": "REQUIRED"
}, {
	"name": "count",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "total",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "flag_1",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "balue_1",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "flag_2",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "created_at",
	"type": "DATETIME",
	"mode": "NULLABLE"
}, {
	"name": "updated_at",
	"type": "DATETIME",
	"mode": "NULLABLE"
}, {
	"name": "lock_version",
	"type": "INTEGER",
	"mode": "REQUIRED"
}]
```

### Oracle
```json
[{
	"name": "id",
	"type": "INTEGER",
	"mode": "REQUIRED"
}, {
	"name": "class_id",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "group_id",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "valid_date",
	"type": "DATETIME",
	"mode": "REQUIRED"
}, {
	"name": "count",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "total",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "flag_1",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "balue_1",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "flag_2",
	"type": "INTEGER",
	"mode": "NULLABLE"
}, {
	"name": "created_at",
	"type": "DATETIME",
	"mode": "NULLABLE"
}, {
	"name": "updated_at",
	"type": "DATETIME",
	"mode": "NULLABLE"
}, {
	"name": "lock_version",
	"type": "INTEGER",
	"mode": "REQUIRED"
}]
```

**Comment by shinichi-takii** [comment]
fixed v1.1.2
